Assets/HoloToolkit/Input/Scripts/FocusedObjectMessageSender.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using HoloToolkit.Unity;
+
+public class FocusedObjectMessageSender : MonoBehaviour
```

Comment:
This would defeat the purpose of helping everyone write less code by adding a simple way to add keyword recognition to dynamically created objects, because then everyone will need to add a variant of this to their code. Most interesting applications are not static, so we'll see a lot of this.